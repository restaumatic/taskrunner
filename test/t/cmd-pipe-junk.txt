# Tests that we report a sensible error message when unable to parse command in cmd pipe
echo junk >&${_taskrunner_request_pipe}
read -u ${_taskrunner_response_pipe} reply
eval "$reply"
