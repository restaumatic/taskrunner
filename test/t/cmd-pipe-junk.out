-- output:
Invalid command pipe request: "junk"
Error: "Error in $: Failed reading: not a valid json value at 'junk'"
-- exit code: 1
